cmd/crictl/main.go

```diff
@@ -133,8 +134,26 @@ func main() {
 	}
 
 	app.Before = func(context *cli.Context) error {
+		isUseConfig := false
 		configFile := context.GlobalString("config")
-		if configFile == "" {
+		if !context.IsSet("config") {
```

Comment:
This could be simplified as

```go
if _, err := os.Stat(configFile); err == nil {
    isUseConfig = true
}
```

Reply:
done

Reply:
We can not omit the error handling. 
We always show result to user when user self indicate the config file in command line.  
But we should not show result to user when the default config file is absence. This is a normal case.